hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -68,6 +69,11 @@
    */
   private long totalWriteErrors;
 
+  /**
+   * Relative path to the temporary file from the base path.
+   */
+  private String tempPath;
```

Comment:
have you tested that this is backwards compatible with current json/hoodie commit metadata?